src/CTA.FeatureDetection.ProjectType/Constants.cs

```diff
@@ -31,6 +31,10 @@ public class Constants
         public const string WCFServiceCodeFeatureName = "CoreWCFServiceCodeFeature";
         public const string WCFClientFeatureName = "WCFClientFeature";
         public const string WCFServiceHostFeatureName = "WCFServiceHostFeature";
+        public const string VBClassLibraryFeatureName = "VBClassLibraryFeature";
+        public const string VBWebApiFeatureName = "VBWebApiFeature";
+        public const string VBWebFormsFeatureName = "VBWebFormsFeature";
```

Comment:
Should we be using nameof(VBWebFormsFeature) for these? Maybe don't even put them in the constants file and just use nameof(VBWebFormsFeature) where they are needed.

Reply:
good point, but making them consistent with existing enums